CHANGELOG.md

```diff
@@ -39,6 +39,7 @@
 * [CHANGE] Ruler: gRPC message size default limits on the Ruler-client side have changed: #3523
   - limit for outgoing gRPC messages has changed from 2147483647 to 16777216 bytes
   - limit for incoming gRPC messages has changed from 4194304 to 104857600 bytes
+* [CHANGE] HA Tracker: verify label value limit for replica label. #3668
```

Comment:
```suggestion
* [CHANGE] HA Tracker: verify label value length limit (`-validation.max-length-label-value`) for replica label. #3668
```

Reply:
Done. I've also moved entry to "master" section, somehow it was in 1.6 :)